README.md

```diff
@@ -106,40 +106,41 @@ Benchmarks measure memoized value retrieval time using
 [`benchmark-ips`](https://github.com/evanphx/benchmark-ips). Benchmarks are
 run in GitHub Actions and updated in every PR that changes code.
 
-**Values >1.00x represent how much _slower_ each gem’s memoized value retrieval
-is than the baseline.**
+All benchmarks are relative to MemoWise.
```

Comment:
I would vote to use language similar to what we had before, and cut this sentence and then say:

```
**Values >1.00x represent how much _slower_ each gem’s memoized value retrieval is than the latest commit of `MemoWise`.**
```